### ISSUE TITLE
Map Pop Up Boxes

### ISSUE THREAD

**Issue by caadler** [uno-isqa-8950/uno-cpi#874]
In the pop-ups, please make the following changes (to the order and variables)
-	Community Partner
-	Number of Projects
-	City (instead of county): 
-	Legislative District: [note the relabel, it should be legislative district and not legislative district number – also, could we remove this variable for non-Nebraska addresses – if not, remove completely]
-	Mission Areas [ensure that there are spaces between multiple mission areas]
-	Community Organization Type [please note this is a relabel – should be reflected throughout]
-	Campus Partner [ensure that there are spaces between multiple campus partners]
-	Academic Year [ensure that there are spaces between multiple academic years]
-	Website [this should be a hyperlink but is not right now] 
![image](https://user-images.githubusercontent.com/35402227/54655843-5c412f80-4a91-11e9-89f1-908682b2efd9.png)

**Comment by vbhat0891** [comment]
This issue is fixed, only one pop up box opens at a time. The contents have been updated per the request. I removed the Legislative district from Infowindow as writing a logic to display this field only for the markers in Nebraska is not worth the time.

Here is a snip of the updated Legislative district map
![image](https://user-images.githubusercontent.com/26983338/54882500-ab3beb80-4e28-11e9-9bec-6c24b675c3ae.png)
